contrib/osx/keybindings.el

```diff
@@ -7,14 +7,17 @@
 
   ;; Keybindings
   (global-set-key (kbd "s-q") 'save-buffers-kill-terminal)
-  (global-set-key (kbd "s-v") 'yank)
-  (global-set-key (kbd "s-c") 'kill-ring-save)
+  (global-set-key (kbd "s-v") 'evil-paste-after)
+  (global-set-key (kbd "s-c") 'evil-yank)
+  (global-set-key (kbd "s-a") 'mark-whole-buffer)
   (global-set-key (kbd "s-x") 'kill-region)
   (global-set-key (kbd "s-w") 'kill-this-buffer)
-  (global-set-key (kbd "s-z") 'undo-tree-undo)
```

Comment:
Thanks for reviewing my PR @trishume.

From what I understand, spacemacs sets `global-undo-tree-mode` (https://github.com/syl20bnr/spacemacs/blob/develop/spacemacs/packages.el#L2695), so regular `undo` behaves like `undo-tree-undo`. 

I personally disable undo-tree-mode in my spacemacs, so undo behaves as expected. 

Setting the `s-z` keybinding to `undo` will make it behave as expected regardless of whether you like undo-tree-mode

Thanks!